src/util.rs

```diff
@@ -23,9 +15,25 @@ pub fn validate_or_format_semver(version: &str) -> Option<String> {
     }
 }
 
+pub fn load_env() {
```

Comment:
we should only load `.env`; and in readme state an explicity copy from `.env.local` -> `.env`.
Saying this because it is a widely used practice (maybe not so much in rust though).

Reply:
The way I set it up is to read from `.env` and override any values with `.env.local`, if present. The nice thing about this is that we can `.gitignore` the local env file, so users can make changes there and not worry about accidentally checking them in. If we have users copy into `.env`, they might accidentally commit in the changes. Good call on updating the readme, I will do that.

Reply:
Standard practice is what is discussed.
However, your reasoning is also sound and does make more sense for us in this case (also makes it easier).